site/src/atomics/border.md

```diff
@@ -118,3 +118,20 @@ The color of a border can be modified using the following classes.
 	<p class="margin-inline-md">Custom Color</p>
 </div>
 ```
+
+### Gradient colors
+
+The gradient colors of a border can be modified using the following classes.
+
+| modifiers | color name         | direction |
+| --------- | ------------------ | --------- |
+| values    | `argon`, `sunrise` | `inverse` |
+
+```html
+<div class="border-color-argon border-md padding-block-md border-radius-lg margin-bottom-xs">
+	<p class="margin-inline-md">Gradient Color</p>
```

Comment:
Nit to align the example text with the other examples:

```suggestion
	<p class="margin-inline-md">Border with gradient</p>
```

Reply:
Updated